Dockerfile

```diff
@@ -14,7 +14,9 @@ COPY . .
 RUN mkdir -p $GOPATH/pkg/linux_amd64/github.com/iotexproject/ && \
     make clean build-all
 
-FROM debian:stretch
+FROM alpine:latest
+
+RUN apk add --no-cache ca-certificates
```

Comment:
this cannot be combined with above?

Reply:
https://docs.docker.com/develop/develop-images/multistage-build/